### ISSUE TITLE
How to handle hosting?

### ISSUE THREAD

**Issue by celsom3** [Cosecha/redadalertas#24]
This is something that hasn't been decided yet. Looking for ideas and suggestions.

The thing to remember is that there are two repos. One serves the API to access the data. The other simply serves the react app. And let's not forget where we want to host the database.

Possible solutions:

- AWS for react client, and server
- Database hosted somewhere else (like in another country)

That's all I have. Suggestions and discussion is welcome here.

**Comment by Sag0Sag0** [comment]
The option to host in another country seems like the most secure option. But it costs money. You could you run a Patreon account to pay for the rent of the server?

**Comment by celsom3** [comment]
Yeah @Sag0Sag0, we would have to pay either way. Any idea of good offshore hosting providers?

**Comment by Sag0Sag0** [comment]
After looking around most of the clouds seem very unsafe. I'd suggest getting hiring a box in Iceland (Icelands parliament is filled with party's like the pirate party.) with ssh access and just setting up there. The one server host i can find with a good track history is https://www.orangewebsite.com/.
